src/main/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusValidation.java

```diff
@@ -299,7 +318,26 @@ private static void validateTableValue(
                 }
                 break;
         }
+    }
+
+    /** Validate a single route_id value for the directions.txt GTFS+ table. */
+    private static void validateTableValue(
+            Collection<ValidationIssue> issues,
+            String tableId,
+            int rowIndex,
+            String[] allValues,
+            String value,
+            JsonNode[] specFieldsFound,
+            JsonNode specField,
+            GTFSFeed gtfsFeed,
+            Map<String, Route> gtfsRoutes
+    ) {
+        if (specField == null) return;
+        validateTableValue(issues, tableId, rowIndex, allValues, value, specFieldsFound, specField, gtfsFeed);
 
+        if (!gtfsRoutes.containsKey(value)) return;
```

Comment:
This if statement is not needed. The remove method does this check before removal.

Reply:
Good call, updated in https://github.com/ibi-group/datatools-server/pull/559/commits/cf83716376c894677cddf3752eb439962f1f4f62